content/docs/components-and-props.md

```diff
@@ -152,7 +152,7 @@ function Comment(props) {
 }
 ```
 
-[](codepen://components-and-props/extracting-components)
+**[Try it on CodePen](https://codepen.io/gaearon/pen/VKQwEo?editors=1010)**
```

Comment:
Code Preview: https://reactjs-org-i5p5km1p8-fbopensource.vercel.app/docs/components-and-props.html
Should I remove this line?
Codepen link we have is of complete example, and till this stage, we are having half code, so it will be better if we remove these codepen link.
Any thoughts?